src/event/fd.rs

```diff
@@ -24,9 +24,18 @@ impl FileDesc {
     /// Set up performance monitoring for
     /// configured event without any flags.
     /// Panics if `perf_event_open()` fails.
-    pub fn new(event: &mut perf_event_attr, pid: i32, cpu: i32, group_fd: i32) -> Self {
+    pub fn new(
+        event: &mut perf_event_attr,
+        child: Option<&std::process::Child>,
```

Comment:
Wondering if this is maybe too specific, because in this case, we can only set up events for child processes, instead of any arbitrary process.